api/community/selectors.py

```diff
@@ -10,9 +11,13 @@ def get_thread(*, id):
     return Thread.objects.get(id=id)
 
 
-def get_recent_comments():
-    # TODO
-    return Comment.objects.select_related("profile__user").with_counts().all()
+def get_recent_comments(days_back=7):
+    today = date.today()
+    recent = today - timedelta(days=days_back)
+    tomorrow = today + timedelta(days=1)
```

Comment:
did you use [recent, tomorrow] instead of [recent, today]  because range query is not inclusive on the end?

Reply:
just investigated how Django does it: https://docs.djangoproject.com/en/3.1/topics/i18n/timezones/#naive-and-aware-datetime-objects

the db insert uses UTC, so I updated `recent` to be 7 days back in UTC, and we can compare timezones since now is one now 💯